### ISSUE TITLE
Subdivide LedMatrix to display different texts

### ISSUE THREAD

**Issue by Surbyte** [AaronLiddiment/RGBLEDS#16]
It's not really a problem, your library is great. 
Is it possible to subdivide a Led Matrix objetc initilization to show different texts?
Imagine 4 divitions for instance: 
time in a corner upper right with a small font. 
temperatura on bottom right with larger font.
Text1 on Upper left with a large font.
counter on the bottom left with small font.

**Comment by AaronLiddiment** [comment]
Firstly, please change over and use the individual libraries rather than this combined one which is older.

To answer your question then yes it should be quite easy.
Just use one cLEDMatrix and multiple cLEDText classes. Each of the text classes should be passed the same Matrix definition.
Then when you initialise each Text class you just assign its area on the Matrix and can use different fonts with each one.
Then you would just call UpdateText member function for each the Text Classes before calling a Fastled show.

**Comment by Surbyte** [comment]
Thank you very much, i will try to do it.

**Comment by Surbyte** [comment]
You are right, too easy, it works!! Two objects cLed with diferents X,Y coordinates. 
Thank you very much.
